documentation/content/howtocontribute.rst

```diff
@@ -196,4 +196,15 @@ We have a simple rule for making a release: It'll be Done when it's done. If
 you want to follow our release plans, just take a look at the milestones_ since
 we use it for self-organization.
 
+Handling Signals
```

Comment:
Now I'm not sure if we should keep this documentation in the `howtocontribute.rst` file or in the `codestyle.rst` file.

Reply:
I think it should be in codestyle. I think `howtocontribute.rst` should describe aspects of the development workflow and codestyle should describe how the code is supposed to look like and behave. With this reasoning in mind, I changed this to `codestyle.rst` in the last update